app/actions/sessions.js

```diff
@@ -364,8 +364,7 @@ export function setConnectionNeedToBeSaved(tabId, bool) {
 function PREVIEW_QUERY (dialect, table, database = '') {
     switch (dialect) {
         case DIALECTS.IBM_DB2:
-            console.warn('IBM', table);
-            return `SELECT * FROM ${table} FETCH FIRST 5 ROWS ONLY`;
+            return `SELECT * FROM ${table}`;
```

Comment:
should we remove the `limit 5` from the rest of the queries too?

Reply:
set limit to 1000. see comments below